arviz/plots/energyplot.py

```diff
@@ -113,11 +112,10 @@ def plot_energy(
                 bw=bw,
                 label=label,
                 textsize=xt_labelsize,
-                plot_kwargs=plot_kwargs,
                 fill_kwargs=fill_kwargs,
+                plot_kwargs=plot_kwargs,
                 ax=ax,
```

Comment:
There should be a `legend=False`, otherwise, the legend of the kde will always be present.